pkgs/servers/x11/xorg/xwayland.nix

```diff
@@ -1,37 +1,31 @@
 { lib, wayland, wayland-protocols, xorgserver, xkbcomp, xkeyboard_config
-, epoxy, libxslt, libunwind, makeWrapper, egl-wayland
-, defaultFontPath ? "" }:
-
-with lib;
+, epoxy, libtirpc, libxslt, libunwind, makeWrapper, egl-wayland, fetchurl, meson, ninja
+, pkg-config, fontutil, defaultFontPath ? "" }:
 
 xorgserver.overrideAttrs (oldAttrs: {
 
-  name = "xwayland-${xorgserver.version}";
-  buildInputs = oldAttrs.buildInputs ++ [ egl-wayland ];
+  name = "xwayland-21.1.0";
```

Comment:
New packages should contain pname + version.

Reply:
https://github.com/NixOS/nixpkgs/pull/115552#discussion_r591115800